File: IPL_TICKET_BOOKING_SYSTEM_INTADM21DN003_POD1_AES_Project_14-master/IPLAngular/src/app/shared/stadium-details.service.ts
import { Injectable } from '@angular/core';
import { HttpClient } from '@angular/common/http';

@Injectable({
  providedIn: 'root'
})
export class StadiumDetailsService {
  readonly url="http://localhost:51985/Api/StadiumDetails/Add_StadiumDetails";
  constructor(private http:HttpClient) { }
  addstadium(value:any){
    const details={
    
      Location:value.Location,
      Name:value.Name,
      Capacity:value.Capacity,
      Home_To:value.Home_To,
      Ends:value.Ends
    }
    console.log(details);
    //return data;
   return this.http.post(this.url,details);
  }
}

File: IPL_TICKET_BOOKING_SYSTEM_INTADM21DN003_POD1_AES_Project_14-master/IPL_TBS.BusinessLogic/StadiumDetailsService.cs
using IPL_TBS.DataAccess.Entities;
using System;
using System.Collections.Generic;
using System.Data.Common;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using IPL_TBS.DataAccess;

namespace IPL_TBS.BusinessLogic
{
	public class StadiumDetailsService : IDisposable
	{
		private readonly RegisterDBContext context;
		public StadiumDetailsService()
		{
			context = new RegisterDBContext();
		}
		public void Dispose()
		{
			context.Dispose();
		}

		public void Post(string Name, string Location, string Capacity, string Home_To, string Ends)
		{
			try
			{
				StadiumDetails obj1 = new StadiumDetails();
				obj1.Name = Name;
				obj1.Location = Location;
				obj1.Capacity = Capacity;
				obj1.Home_To = Home_To;
				obj1.Ends = Ends;

				context.StadiumDetail.Add(obj1);
				context.SaveChanges();
			}
			catch (DbException ex)
			{
				throw new IPLException("Error Reading Data", ex);

			}
			catch (Exception ex)
			{
				throw new IPLException("Unknown Error while reading Stadiumdetails", ex);

			}
		}
	}
}